lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -535,12 +541,11 @@ def _validate_retry_presubmit(self, job_id: str, as_admin: bool = False):
         job = self.sdkmr.get_job_with_permission(
             job_id, JobPermissions.WRITE, as_admin=as_admin
         )  # type: Job
-        job_input = job.job_input  # type: JobInput
 
-        parent_job = None
-        if job_input.parent_job_id:
-            parent_job = self.sdkmr.get_job_with_permission(
-                job_input.parent_job_id, JobPermissions.WRITE, as_admin=as_admin
+        batch_parent = None
```

Comment:
Maybe we want to avoid any mention of "parent", since it now implies kbparallels? We could instead do batch_container or batch_id?

Reply:
I changed it to `batch_job`. `batch_id` seems wrong since it's not an ID, and `batch_container` seems a little weird to be as well. It is a job, so `batch_job` seems ok